ui/src/app/pages/workspace/workspace-list.tsx

```diff
@@ -359,8 +359,8 @@ export const WorkspaceList = withUserProfile()
     // Maps each "Filter by" dropdown element to a set of access levels to display.
     const filters = [
       { label: 'Owner',  value: ['OWNER'] },
-      { label: 'Writer', value: ['OWNER', 'WRITER'] },
-      { label: 'Reader', value: ['OWNER', 'READER'] },
+      { label: 'Writer', value: ['WRITER'] },
+      { label: 'Reader', value: ['READER'] },
```

Comment:
Note: Just followed the thread on this when seeing this in prod - the resolution here looks correct to me. But wanted to clarify a point: owner, writer, and reader are concentric roles in our system. As in, a "Writer" can also "Read" (i.e. in the initial implementation, I would have expected `Reader -> ['OWNER', 'WRITER' 'READER']`).